packages/Chem/src/tests/projects-tests.ts

```diff
@@ -0,0 +1,363 @@
+    molecules: tv.dataFrame.col('smiles'),
+    methodName: 'UMAP',
+    similarityMetric: 'Tanimoto',
+    plotEmbeddings: true,
+    options: undefined,
+    preprocessingFunction: undefined,
+    clusterEmbeddings: true
+  }).call(undefined, undefined, { processed: false });
+  //need for scatter plot to render
+  await delay(10);
+}
+
+async function runStructuralAlerts(tv: DG.TableView): Promise<void> {
+  await DG.Func.find({ package: 'Chem', name: 'runStructuralAlerts' })[0].prepare({
+    table: tv.dataFrame,
+    molecules: tv.dataFrame.col('smiles'),
+    pains: true,
+    bms: false,
+    sureChembl: false,
+    mlsmr: false,
+    dandee: false,
+    inpharmatica: false,
+    lint: false,
+    glaxo: false
+  }).call(undefined, undefined, { processed: false });
+}
+
+async function runElementalAnalysis(tv: DG.TableView): Promise<void> {
+  await DG.Func.find({ package: 'Chem', name: 'elementalAnalysis' })[0].prepare({
+    table: tv.dataFrame,
+    molecules: tv.dataFrame.col('smiles'),
+    radarViewer: true,
+    radarGrid: true
+  }).call(undefined, undefined, { processed: false });
+}
+
+async function runConvertNotation(tv: DG.TableView): Promise<void> {
+  await DG.Func.find({ package: 'Chem', name: 'convertNotation' })[0].prepare({
+    data: tv.dataFrame,
+    molecules: tv.dataFrame.col('smiles'),
+    targetNotation: 'molblock',
+    overwrite: false,
+    join: true
+  }).call(undefined, undefined, { processed: false });
+}
+
+async function runNamesToSmiles(tv: DG.TableView): Promise<void> {
+  await DG.Func.find({ package: 'Chem', name: 'namesToSmiles' })[0].prepare({
+    data: tv.dataFrame,
+    names: tv.dataFrame.col('Name'),
+  }).call(undefined, undefined, { processed: false });
+}
+
+async function runCurate(tv: DG.TableView): Promise<void> {
+  const df = await grok.functions.call(`Chem:Curate`, {
+    data: tv.dataFrame,
+    molecules: tv.dataFrame.col('smiles'),
+    kekulization: true,
+    normalization: false,
+    reionization: false,
+    neutralization: false,
+    tautomerization: false,
+    mainFragment: false,
+  });
+  tv.dataFrame.columns.add(df.col('curated_molecule'));
+}
+
+async function runAddChemPropertiesColumns(tv: DG.TableView): Promise<void> {
+  await DG.Func.find({ package: 'Chem', name: 'addChemPropertiesColumns' })[0].prepare({
+    table: tv.dataFrame,
+    molecules: tv.dataFrame.col('smiles'),
+    MW: true,
+    HBA: false,
+    HBD: false,
+    logP: false,
+    logS: false,
+    PSA: false,
+    rotatableBonds: false,
+    stereoCenters: false,
+    moleculeCharge: false,
+  }).call(undefined, undefined, { processed: false });
+}
+
+async function runToInchiKeys(tv: DG.TableView): Promise<void> {
+  await DG.Func.find({ package: 'Chem', name: 'addInchisKeysTopMenu' })[0].prepare({
+    table: tv.dataFrame,
+    molecules: tv.dataFrame.col('smiles'),
+  }).call(undefined, undefined, { processed: false });
+}
+
+async function runToInchi(tv: DG.TableView): Promise<void> {
```

Comment:
What is this? Why do we have all these wrappers that do nothing but call a function?

Reply:
If I remember correctly, I need the fucntion to be called with 'processed: false' parameter. But if I call function directly, processed parameter is set to true.